### DISCUSSION TITLE
Kotlin models to have default value null when nulable

### DISCUSSION THREAD

**Discussion by anloboda** [kobylynskyi/graphql-java-codegen#1249] [Q&A] [answered]
Is it possible to configure plugin to generate default value for model parameters?

```
data class User(
   var firstName,
   var lastName,
   var middleName? = null
)
```

**Comment by kobylynskyi** [comment, marked as answer]
Hi @anloboda 
Default values are driven by the schema only. So if it is present in the schema then you will see this in the generated classes.
Hope this helps.
